docs/Contributing/API-for-contributors.md

```diff
@@ -454,7 +454,7 @@ If the `name` is not already associated with an existing team, this API route cr
 | Name          | Type   | In   | Description                                                                                                                                                                                                                                             |
 | ------------- | ------ | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | name          | string | body | **Required.** The team's name.                                                                                                                                                                                                                          |
-| agent_options | string | body | **Required.** The agent options spec that is applied to the hosts assigned to the specified to team. These agent agent options completely override the global agent options specified in the [`GET /api/v1/fleet/config API route`](#get-configuration) |
+| agent_options | string | body | **Required.** The agent options spec that is applied to the hosts assigned to the specified to team. These agent agent options completely override the global agent options specified in the [`GET /api/v1/fleet/config API route`](#get-configuration)  |
```

Comment:
"The agent options spec that is applied to the hosts assigned to the specified to team. These agent agent options completely override the global agent options specified in the" needs to be rewritten for clarity in completed sentences.

Reply:
> These agent agent options completely override the global agent options specified in the `GET /api/v1/fleet/config API route`.

@Desmi-Dizney is the above sufficient? This is how the current sentence reads without the special ()[] markdown link formatting.